core/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -220,6 +220,13 @@ public boolean hasIndexOrAlias(String aliasOrIndex, ClusterState state) {
     }
 
     /**
+     * @return If the specified string is data math expression then this method returns the resolved expression.
+     */
+    public String resolveDateMathExpression(String dateExpression) {
+        return dateMathExpressionResolver.resolveExpression(dateExpression, new Context(null, null));
```

Comment:
the initialization of the context with two null arguments makes me wonder why we don't need those arguments there.